tests/frontend/matcher/test_matcher_base.py

```diff
@@ -53,16 +53,21 @@ def __assert_one_to_one_constraint(self, match_idxs: np.ndarray):
     def test_empty_input(self):
         """Tests the matches when there are no descriptors."""
 
-        nonempty_keypoints, _, nonempty_descriptors, _, = generate_random_input()
+        nonempty_keypoints, _, nonempty_descriptors, _, _, _ = generate_random_input()
         empty_keypoints = Keypoints(coordinates=np.array([]))
         empty_descriptors = np.array([])
+        im_shape = (300, 200)
```

Comment:
maybe create two variables im1_shape and im2_shape to prevent confusion?

Reply:
sure, updated now